### PR TITLE
Introduce click to selection station function

### LINKED ISSUE #53: Click to select station and associated results
The user should be able to click on a station to select it. This would filter the data in the grid to show only the data associated with that station while still honoring any existing filters. The user could then click on the map (not on a station) to clear the selection or click on a "clear selection" button on the grid. The "clear selection" button would only appear if a station is selected.

@inkenbrandt: Do we need to allow for multiple stations to be selected?

### PR DESCRIPTION
**Pull request by stdavis**
Closes #53

### CODE REVIEW

**Comment by stdavis** [comment]
@inkenbrandt This has been pushed to the test server. Would you mind testing it out and letting me know what you think?

**Comment by inkenbrandt** [comment]
I would Love to!

**Comment by inkenbrandt** [comment]
Looks Good Scott!

**Comment by stdavis** [comment]
@steveoh This is ready for a review if you are interested.

File: src/app/mapController.js [hunk 1]
```diff
@@ -75,13 +86,18 @@ define([
             this.fLayer.on('load', function () {
                 that.fLayer.renderer.symbol.setSize(config.stationSymbolSize);
             });
+            this.fLayer.on('click', function selectFeatureLayerStation(evt) {
+                that.clearStationSelection();
+                that.selectStation(evt.graphic);
+                evt.stopPropagation();
```
Comment (steveoh) [inline src/app/mapController.js]: do you need preventDefault here as well?

Comment (stdavis) [inline src/app/mapController.js]: I don't think so. What default would I be preventing? I just want to stop the click event from bubbling up to the map.

Comment (steveoh) [inline src/app/mapController.js]: I suppose if we only support ie 9+ this is ok.

**Pull request merged**